dali/aux/optical_flow/turing_of/optical_flow_turing.cc

```diff
@@ -86,10 +87,28 @@ void OpticalFlowTuring::CalcOpticalFlow(
         dali::kernels::TensorView<dali::kernels::StorageGPU, const uint8_t, 3> input_image,
         dali::kernels::TensorView<dali::kernels::StorageGPU, float, 3> output_image,
         dali::kernels::TensorView<dali::kernels::StorageGPU, const float, 3> external_hints) {
```

Comment:
Maybe you can shorten this `dali::kernels::TensorView<dali::kernels::StorageGPU...` by placing this in appropraite namespace or just `using ...`?